VK_V1.0/VK_V1.0/Animations/cloudView.swift

```diff
@@ -0,0 +1,127 @@
+// cloudView.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+/// Облако
+final class CloudView: UIView {
+    // MARK: - Private IBOutlet
+
+    @IBOutlet private var cloudView: UIView!
+    @IBOutlet private var backView: UIView!
+
+    // MARK: - Private property
```

Comment:
```suggestion
    // MARK: - Private Visual Components
```
кажется больше относится к визуальным компонентам